docs/data/joy/components/autocomplete/autocomplete.md

```diff
@@ -0,0 +1,355 @@
+---
+product: joy-ui
+title: React Autocomplete component
+githubLabel: 'component: autocomplete'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/
+---
+
+# Autocomplete
+
+<p class="description">The autocomplete is a normal text input enhanced by a panel of suggested options.</p>
+
+The widget is useful for setting the value of a single-line textbox in one of two types of scenarios:
```

Comment:
This intro feels very disconnected⎯it got me confused at first.

Reply:
It has been rewritten, please take a look again. I think @samuelsycamore can refine this at the end.